battles/models.py

```diff
@@ -32,7 +32,10 @@ def __str__(self):
 
 
 class TrainerTeam(models.Model):
-    trainer = models.ForeignKey(User, on_delete=models.CASCADE)
+    trainer = models.ForeignKey(
+        User, on_delete=models.CASCADE,
+        related_name="teams"
+    )
     pokemon_1 = models.ForeignKey(
         Pokemon, on_delete=models.CASCADE,
         related_name="pokemon_1",
```

Comment:
The same thing applies for the `related_name`s in this whole model. Please check all your models and do the appropriate changes to the related names (either change the names or put a `'+'` to not have it created) :)

Reply:
Done!